### DISCUSSION TITLE
[ESP lora device] How login to this dev trought wifi?

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#221] [Help & Questions] [answered]
I put firmware to my device ` $rnodeconf --autoinstall`
I connect it to powerbank. How connect to it from other host from WiFi? I disconect my esp device from computer. I need it on the roof.

How discovering other nodes are in my network or not?

**Comment by Swissbandit** [comment]
https://unsigned.io/guides/2023_01_19_RNode_Bluetooth_Setup_In_Sideband_On_Android.html
and
https://unsigned.io/guides/2022_03_26_private-messaging-over-lora.html

**Comment by dmahony** [comment]
They are asking how to set the RNode device as a WiFi client. I am also hoping to do this.

**Reply by markqvist** [reply]
Not currently possible. Host connections only work over Bluetooth, USB or serial port at the moment.

**Comment by cobraPA** [comment, marked as answer]
You would need to add another host device presently, so linux/android/windows device with serial connection to RNode on the roof.  Linux device can join the wifi and bridge to RNode (using Reticulum on the Linux device.)

**Reply by fab1an2** [reply]
esp have wifi why using another host?

**Reply by cobraPA** [reply]
The RNode firmware does not support Wifi or running Reticulum (the network stack) currently.  You could look at this: https://github.com/markqvist/Reticulum/discussions/21 for the work by atterman on the ESP32 C port.  It sounds like that could run Lora and Reticulum on ESP32.  It is not done yet but may be usable now.
